lib/fs.js

```diff
@@ -196,57 +196,75 @@ function Stats(
   this.ino = ino;
   this.size = size;
   this.blocks = blocks;
-  this._atim_msec = atim_msec;
-  this._mtim_msec = mtim_msec;
-  this._ctim_msec = ctim_msec;
-  this._birthtim_msec = birthtim_msec;
+  this.atimeMs = atim_msec;
+  this.mtimeMs = mtim_msec;
+  this.ctimeMs = ctim_msec;
+  this.birthtimeMs = birthtim_msec;
+  this._atime = null;
+  this._mtime = null;
+  this._ctime = null;
+  this._birthtime = null;
 }
 fs.Stats = Stats;
 
 // defining the properties in this fashion (explicitly with no loop or factory)
 // has been shown to be the most performant on V8 contemp.
+// TODO(refack): re-evaluate when TF+I lands
```

Comment:
Why this addition? In general *many* things in node will need to be re-evaluated when V8 6.0 is merged in, so singling out one particular possible instance doesn't really help much. \*shrug\*